LEGO1/legoinputmanager.h

```diff
@@ -15,7 +15,7 @@ __declspec(dllexport) enum NotificationId
 
 // VTABLE 0x100d8760
 // SIZE 0x338
-class LegoInputManager
+class LegoInputManager : public MxPresenter
```

Comment:
Can you make sure the `LegoInputManager` structure is still functional after the addition of this base class? I'm assuming you'd have to adjust the private member padding. `ISLE.EXE` depends on a few members in here.

Reply:
Can you give https://github.com/isledecomp/isle/pull/59 another look?
It adds compile time assertions to quit a few classes.

If MxPresenter has size 64, I'll subtract 60 from the top member.

Reply:
Oh, it's already merged.

Reply:
I think it was an accident (or a glitch?) - but I think as long as we fix the structure of `LegoInputManager` it should be fine

Reply:
I think the current size is of by 4 bytes.
When you add all members, you get `0x19c+4+0x194+4` = 824. Including the vtable, this becomes 828 bytes.
But the OG dll allocates 824 bytes.
I'm going to remove 4 bytes from the second array in #59.

Reply:
The vtable would be "hidden" because it's included as part of the structure MxCore, that's probably why it doesn't add up mathematically

Reply:
Fixed it. `sizeof`  of #59 agrees.